src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -288,7 +289,7 @@ export default class AircraftCommander {
     runSpeed(aircraft, data) {
         const speed = data[0];
```

Comment:
I know this was already there, but could you update this var name to `nextSpeed`?